source/common/router/route_config_update_receiver_impl.cc

```diff
@@ -32,13 +32,25 @@ bool RouteConfigUpdateReceiverImpl::onVhdsUpdate(
     const Protobuf::RepeatedPtrField<envoy::api::v2::Resource>& added_resources,
     const Protobuf::RepeatedPtrField<std::string>& removed_resources,
     const std::string& version_info) {
+  // TODO: check if any changes have been made
```

Comment:
If a WIP TODO, is this ready for implementation? If a real TODO, add (username).

Reply:
Removed.